.appveyor.yml

```diff
@@ -4,13 +4,15 @@ platform:
 #matrix:
 #  fast_finish: true # kills the build at the first failure
 clone_folder: C:\projects\dbal
-clone_depth: 1
+clone_depth: 2
```

Comment:
Is it for `ocular` to be able to determine something? Will it be always enough?

Reply:
Without that the command used to list commit's parents fails: `git log --pretty="%P" -n1 HEAD`
With depth 2 it works, seems to be fine for both single and merge commits (for merge commits it fetches both parents).

```
$ git clone git@github.com:doctrine/dbal --branch=master --depth=2 dbal-shallow -q
$ cd dbal-shallow/
$ git log --oneline 
6ee7d7d (HEAD -> master, origin/master, origin/HEAD) Merge pull request #3117 from morozov/develop-badges
9b7a04a (grafted) Develop badges
5ba43cf (grafted) Merge pull request #3112 from morozov/run-all-cleanup
```